pyat/at/matching/globalfit.py

```diff
@@ -4,91 +4,105 @@
 """
 from typing import Optional
 import numpy
+from warnings import warn
 from ..lattice import Lattice, Refpts, set_value_refpts
+from ..lattice import AtWarning, AtError
```

Comment:
I have "Unused import statement" warnings. Apparently warn is not used… Are these lines left after tests?